packages/common/testing/src/mock_platform_location.ts

```diff
@@ -105,6 +105,7 @@ export const MOCK_PLATFORM_LOCATION_CONFIG =
 export class MockPlatformLocation implements PlatformLocation {
   private baseHref: string = '';
   private hashUpdate = new Subject<LocationChangeEvent>();
+  private _urlChangeIndex: number = 0;
```

Comment:
We can probably remove leading `_` to be consistent with other private properties:
```suggestion
  private urlChangeIndex: number = 0;
```